test/JDBC/input/BABEL-4046-vu-verify.sql

```diff
@@ -35,6 +35,6 @@ GO
 	select DM.empno, DM.ename, DM.baseloc from EMP_T DM where DM.baseloc in (select baseloc from t4)  order by DM.empno
 GO
 
-
-
-
+;with EMP_T AS ( select empno, ename, CASE WHEN baseloc LIKE 'AUS%' THEN REPLACE(baseloc,'AUSTIN','A') ELSE baseloc END AS baseloc, deptno from t3)
```

Comment:
We should add tests for cases when the left and right arg is different node as well. For example:
1. LA - Const, RA - ColumnRef
2. LA and RA - Const
3. LA and RA - ColumnRef
4. What is the behaviour if  CASE WHEN is wrapped inside another CASE WHEN with LIKE

Such variations needs to be tested.

Reply:
The issue is not happening inside case when, the operator not get transformed is actually '=' , so I added the args transform method for those operators.
For case when transform, the implement is inside : pgtsql_expression_tree_mutator function. And also, those test cases are actually already in other test cases files.

Reply:
@forestkeeper I agree but first three cases seems relevant here. @ahmed-shameem can you please provide example also?

Reply:
As discussed offline, we should add tests for other OpExpr like <,>,<=,>=. We should also test for this case DM.baseloc1 = DM.baseloc2